frontend/src/components/layout/Header/Header.styled.ts

```diff
@@ -45,15 +66,41 @@ export const BlueCasperLogo = styled.img`
   }
 `;
 
-export const HeroContainer = styled.div`
+export const LogoText = styled.span`
+  font-family: lausanne;
+  font-size: ${pxToRem(26)};
+  font-weight: ${fontWeight.medium};
+  letter-spacing: ${pxToRem(-1)};
+  padding-top: ${pxToRem(6)};
+  position: absolute;
+  left: ${pxToRem(119)};
```

Comment:
Hmm, this seems a bit fishy

Reply:
Will look into it!

Reply:
Ahh okay, so we will be removing these styles once we have the updated image